### PR TITLE
Debug for empty error str to KubeClusterUpdateStatus

### PR DESCRIPTION
**Pull request by andrewd-zededa**
And reduce some other verbose logging.

### CODE REVIEW

**Review by naiming-zededa — APPROVED** [review]
LGTM

File: pkg/pillar/cmd/zedagent/zedagent.go [hunk 1]
```diff
@@ -2117,19 +2117,13 @@ func handleClusterUpdateStatusImpl(ctxArg interface{}, key string,
...
-	//	return
-	//}
-	log.Noticef("handleClusterUpdateStatusImpl devicestate:%s", state)
+	log.Noticef("handleClusterUpdateStatusImpl key:%s obj:%v devicestate:%s", key, req, state)
 	triggerPublishKubeClusterUpdateStatus(ctx)
 }
 
 func triggerPublishKubeClusterUpdateStatusToDest(ctxPtr *zedagentContext, dest destinationBitset) {
-	log.Notice("Triggered PublishKubeClusterUpdateStatus")
+	log.Function("Triggered PublishKubeClusterUpdateStatus")
```
Comment (zedi-pramodh) [inline pkg/pillar/cmd/zedagent/zedagent.go:2126]: Ins't it supposed to be log.Functionf

Comment (andrewd-zededa) [reply pkg/pillar/cmd/zedagent/zedagent.go:2126]: The only argument here is the string so Function should be ok

Comment (zedi-pramodh) [reply pkg/pillar/cmd/zedagent/zedagent.go:2126]: ok

**Pull request merged**